docs/docs-content/enterprise-version/system-management/ssl-certificate-management.md

```diff
@@ -31,41 +38,107 @@ You can swap out the external endpoint certificate at any time without affecting
 
-5. Copy and paste the certificate key into the **Key** field.
+7. Copy and paste the certificate authority into the **Certificate Authority** field.
 
-6. Copy and paste the certificate authority into the **Certificate authority** field.
+   ![A view of the certificate upload screen](/palette_system-management_ssl-certificate-management_system-address.webp)
 
-   ![A view of the certificate upload screen](/palette_system-management_ssl-certifiacte-management_certificate-upload.webp)
+8. Click on **Update** to save your changes.
 
-7. Save your changes.
+Palette validates the combination of system address, certificate, key, and Certificate Authority (CA). Ensure that the
+certificate is not expired, as well as that it is valid for the CA and the system address. Additionally, the system
+address must be accessible from the system console.
 
-If the certificate is invalid, you will receive an error message. Once the certificate is uploaded successfully, Palette
-will refresh its listening ports and start using the new certificate.
+You will receive an error message if the provided values are not valid. Once the certificate is uploaded successfully,
+Palette will refresh its listening ports and start using the newly configured values.
 
-## Validate
+### Validate
 
 You can validate that your certificate is uploaded correctly by using the following steps.
 
-1. Log out of the Palette system console. If you are already logged in, log out and close your browser session. Browsers
-   cache connections and may not use the newly enabled HTTPS connection. Closing your existing browser session avoids
-   issues related to your browser caching an HTTP connection.
+1.  Log out of the Palette system console. If you are already logged in, log out and close your browser session.
+    Browsers cache connections and may not use the newly enabled HTTPS connection. Closing your existing browser session
+    avoids issues related to your browser caching an HTTP connection.
 
-2. Log back into the Palette system console. Ensure the connection is secure by checking the URL. The URL should start
-   with `https://`.
+2.  Log back into the Palette system console. Ensure the connection is secure by checking the URL. The URL should start
+    with `https://`.
 
 Palette is now using your uploaded certificate to create a secure HTTPS connection with external clients. Users can now
 securely access the system console, Palette dashboard, the gRPC endpoint, and the Palette API endpoint.
+
+## Reconcile System Address on Deployed Clusters
+
+Once you have updated your system address, you will need to manually update the API endpoint on each cluster that was
+deployed before the update. This enables the Palette agent to reconnect to the API at the newly configured system
+address.
+
+### Prerequisites
+
+- A tenant. Refer to the [Create a Tenant](./tenant-management.md#create-a-tenant) guide for further information.
```

Comment:
Maybe we don't need this prereq after all? 🤔 Because they can only have clusters deployed when they have a tenant, so it kind of feels redundant. What do you think?